azurerm/internal/services/search/search_service_data_source_test.go

```diff
@@ -0,0 +1,47 @@
+package search_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+)
+
+type SearchServiceDataSource struct {
+}
+
+func TestAccDataSourceSearchService_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_search_service", "test")
+	r := SearchServiceDataSource{}
+
+	data.DataSourceTest(t, []resource.TestStep{
+		{
+			Config: r.basic(data),
+			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).Key("id").Exists(),
```

Comment:
we can remove this, since otherwise the resource won't be in the state/will raise an error (which'll be hit before this point)

```suggestion
```